script.js

```diff
@@ -46,29 +57,25 @@ function addBook() {
            document.getElementById('book-bd').style.visibility = "visible";
            //document.getElementById('form').style.opacity = "0";
            document.getElementById('book-save').style.visibility = "hidden";
```

Comment:
you can wrap this code in function showSuccessStatus() 
call it and call cleanFormInput()

Reply:
and move this part to the end of function